kratos/tests/test_compare_elements_conditions.py

```diff
@@ -0,0 +1,37 @@
+from __future__ import print_function, absolute_import, division
+
+import KratosMultiphysics.KratosUnittest as KratosUnittest
+import KratosMultiphysics as KM
+
+class TestCompareElementsAndConditionsUtility(KratosUnittest.TestCase):
+
+    def test_compare_elements(self):
+        current_model = KM.Model()
+
+        model_part= current_model.CreateModelPart("Main")
+
+        model_part.CreateNewNode(1, 0.00,0.00,0.00)
+        model_part.CreateNewNode(2, 1.00,0.00,0.00)
+        model_part.CreateNewNode(3, 1.00,1.00,0.00)
+        model_part.AddProperties(KM.Properties(1))
+        pelem = model_part.CreateNewElement("Element2D3N", 1, [1,2,3], model_part.GetProperties()[1])
+
+        self.assertEqual(KM.CompareElementsAndConditionsUtility.GetRegisteredName(pelem), "Element2D3N")
+        self.assertNotEqual(KM.CompareElementsAndConditionsUtility.GetRegisteredName(pelem), "NotElement2D3N")
+
+    def test_compare_conditions(self):
+        current_model = KM.Model()
+
+        model_part= current_model.CreateModelPart("Main")
+
+        model_part.CreateNewNode(1, 0.00,0.00,0.00)
+        model_part.CreateNewNode(2, 1.00,0.00,0.00)
+        model_part.CreateNewNode(3, 1.00,1.00,0.00)
+        model_part.AddProperties(KM.Properties(1))
+        pcond = model_part.CreateNewCondition("Condition3D", 1, [1,2,3], model_part.GetProperties()[1])
```

Comment:
this is deprecated and will make transition more difficult, can you please change it?

Reply:
Because we have not removed the old name when it looks in the database (which is alphabetically ordered) it fins the deprecated one first. We should change this at the same time than the deprecation, not before

Reply:
ok